_format/1.1/index.md

```diff
@@ -360,10 +372,21 @@ response that includes the resource as the primary data.
 A "resource identifier object" is an object that identifies an individual
 resource.
 
-A "resource identifier object" **MUST** contain `type` and `id` members.
+A "resource identifier object" **MUST** contain a `type` member. A resource
+identifier object **MUST** also contain an `id` member, except when it contains
+a `lid` that identifies it as matching a new resource to be created on the
+server.
+
+A "resource identifier object" **MAY** contain a `lid` member. If `id` is
+omitted due to the exception described above, a `lid` member **MUST** be
+included. A `lid` member **MUST** also be included if the referenced resource is
+represented elsewhere in the document with a `lid`. The value of a `lid` member
+**MUST** be identical for every representation of the resource in the document.
+
+The values of the `id`, `lid`, and `type` members **MUST** be strings.
 
-A "resource identifier object" **MAY** also include a `meta` member, whose value is a [meta] object that
-contains non-standard meta-information.
+A "resource identifier object" **MAY** also include a `meta` member, whose value
```

Comment:
There is no actual change here, just reflowing. This change can/should hence be reverted AFAICT.